wolfcrypt/src/siphash.c

```diff
@@ -465,6 +465,12 @@ int wc_SipHash(const unsigned char* key, const unsigned char* in, word32 inSz,
         "orq    %%r12, %%r13\n\t"
         "L_siphash_n6:\n\t"
 
+    : [in] "+r" (in), [inSz] "+r" (inSz)
```

Comment:
Why was this needed?
It is dangerous to split assembly code in the middle like this as the used registers aren't necessarily still the same values.